examples/seldon_deployment/continuous_deployment_config_sklearn.yaml

```diff
@@ -0,0 +1,31 @@
+name: continuous_deployment_pipeline
```

Comment:
for other examples you mentioned in the README the option to run with `-c config`, why not for Seldon?